README.md

```diff
@@ -7,79 +7,87 @@
+KAGOME is a [Polkadot Host](https://github.com/w3f/polkadot-spec/tree/master/host-spec) (former Polkadot Runtime Environment) developed by [Soramitsu](https://soramitsu.co.jp/) and funded by a Web3 Foundation [grant](https://github.com/w3f/Web3-collaboration/blob/master/grants/grants.md).
 
 
 ## Status
 
-Kagome is getting closer to the first official release. Team is currently focused on stabilizing features required to run syncing and validating full nodes.
+- [x] Syncing node
+    - Polkadot, Kusama and Rococo compatibility
+- [x] Validating node
+- [x] Polkadot JS apps support
+- [ ] Parachains support
+- [x] Telemetry support
+- [ ] Fast sync
+- [ ] Warp sync
+- [ ] Light client
 
-A simple status of Kagome development can be found within the [supported features](./README.md/#supported-features) section.
+More details of KAGOME development can be found within the [supported features](./README.md/#supported-features) section and in [projects board](https://github.com/soramitsu/kagome/projects/2)
 
 
 
 ## Getting Started
 
-### Prerequisites
+### Build
+
+#### Prerequisites
 
 For now, please refer to the [Dockerfile](housekeeping/docker/kagome-dev/minideb.Dockerfile) to get a picture of what you need for a local build-environment.
 
-### Build
 
 ```sh
 git clone https://github.com/soramitsu/kagome
 cd kagome
 
-mkdir build && cd build
-cmake -DCMAKE_BUILD_TYPE=Release ..
-make kagome -j $(( $(nproc 2>/dev/null || sysctl -n hw.ncpu) / 2 + 1 ))
+make build
 ```
 
-`make kagome -j <numbers of cores>` (should be less than you have, here it is half of your cores + 1)
+You will get KAGOME binary in the `build/node/` folder
 
+Other make commands are:
 
-### Build with docker
-
-```sh
-git clone https://github.com/soramitsu/kagome
-cd kagome
+```
+make docker
+make command args="gcc --version"
+make release
+make release_docker
+make debug_docker
+make clear
+```
 
-# build and run tests
-INDOCKER_IMAGE=soramitsu/kagome-dev:4-minideb BUILD_DIR=build BUILD_THREADS=9 ./housekeeping/indocker.sh ./housekeeping/make_build.sh
+### Executing KAGOME node
 
-# You can use indocker.sh to run any script or command inside docker
-# It mounts project dir and copy important env variable inside the container.
-INDOCKER_IMAGE=soramitsu/kagome-dev:4-minideb ./housekeeping/indocker.sh gcc --version
+#### Obtaining database snapshot (optional)
 
-## Build Release 
-# Build Kagome
-INDOCKER_IMAGE=soramitsu/kagome-dev:4-minideb BUILD_DIR=build ./housekeeping/indocker.sh ./housekeeping/docker/release/makeRelease.sh
-# Create docker image and push 
-VERSION=0.0.1 BUILD_DIR=build BUILD_TYPE=Release ./housekeeping/docker/kagome/build_and_push.sh
+In order to avoid syncing from scratch we are maintaining the most recent snapshot of Polkadot network for KAGOME node available for anyone here: https://drive.google.com/drive/folders/1pAZ1ongWB3_zVPKXvgOo-4aBB7ybmKy5?usp=sharing
```

Comment:
Maybe add hash of this snapshot?

Reply:
then we will need to update readme more frequently)